openlibrary/plugins/openlibrary/js/lists/ListViewBody.js

```diff
@@ -55,8 +83,9 @@ const getConfirmButtonLabelText = () => {
 };
 
 // Add listeners to each .listDelete link element
-$('.listDelete a').on('click', function() {
-    if (get_seed_count() > 1) {
+// Sometimes .listDelete is dynamically added to the DOM, so we'll add the listener to a parent element
```

Comment:
Why is it that .listDelete is added dynamically to the DOM? It should always be rendered by the server?

Reply:
Originally the change to ListViewBody was to remove the (get_seed_count > 1) statement so that removing the last item from a list didn't prompt the user to delete the whole list. 0c2cc52bf7205a00acb2f160245a447fce011175

The intent of commit 0abc0ad74bd572b714898488a76b4c7ea5c8a2af was to rollback those changes as that statement was recently updated and affirmed. It seems like that rollback didn't go through all the way. Lets make it so ListViewBody.js isn't updated any longer. Thanks!